optaplanner-persistence/optaplanner-persistence-jackson/src/main/java/org/optaplanner/persistence/jackson/impl/domain/solution/JsonSolutionFileIO.java

```diff
@@ -0,0 +1,59 @@
+package org.optaplanner.persistence.jackson.impl.domain.solution;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.optaplanner.core.api.domain.solution.PlanningSolution;
+import org.optaplanner.persistence.common.api.domain.solution.SolutionFileIO;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+/**
+ *
+ * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
+ */
+public class JsonSolutionFileIO<Solution_> implements SolutionFileIO<Solution_> {
```

Comment:
Naming wise, let's use the word Jackson in here, in case we build a Jsonb version too.

Reply:
So JacksonSolutionFileIO? And ...Test.

Reply:
I changed the class **JsonSolutionFileIO** to **JacksonSolutionFileIO** which is much more appropriate.